### ISSUE TITLE
Javascript bundle fails to bundle signalr.js file

### ISSUE THREAD

**Issue by afranz33** [trullock/NUglify#298]
When I try using this pipeline code pipeline.AddJavaScriptBundle and adding the signalr.js (latest dist) file it fails with error stack of below. I tried bundling it by itself, adding it with others etc and get the same issue.

message:
Exception thrown at 0x00007FFD7C144F69 (KernelBase.dll) in w3wp.exe: 0xC0000005: Access violation reading location 0x0000000000000000.

Stack
System.NullReferenceException: Object reference not set to an instance of an object.
   at NUglify.JavaScript.Visitors.NewParensVisitor.Visit(MemberExpression node)
   at NUglify.JavaScript.Syntax.MemberExpression.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.NewParensVisitor.Visit(MemberExpression node)
   at NUglify.JavaScript.Syntax.MemberExpression.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(CallExpression node)
   at NUglify.JavaScript.Syntax.CallExpression.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.AcceptNodeWithParens(AstNode node, Boolean needsParens)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(VariableDeclaration node)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(LexicalDeclaration node)
   at NUglify.JavaScript.Syntax.LexicalDeclaration.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(BlockStatement node)
   at NUglify.JavaScript.Visitors.OutputVisitor.OutputFunctionArgsAndBody(FunctionObject node)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(FunctionObject node)
   at NUglify.JavaScript.Syntax.FunctionObject.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(ClassNode node)
   at NUglify.JavaScript.Syntax.ClassNode.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(BlockStatement node)
   at NUglify.JavaScript.Visitors.OutputVisitor.OutputFunctionArgsAndBody(FunctionObject node)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(FunctionObject node)
   at NUglify.JavaScript.Syntax.FunctionObject.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(GroupingOperator node)
   at NUglify.JavaScript.Syntax.GroupingOperator.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.AcceptNodeWithParens(AstNode node, Boolean needsParens)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(CallExpression node)
   at NUglify.JavaScript.Syntax.CallExpression.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(ReturnStatement node)
   at NUglify.JavaScript.Syntax.ReturnStatement.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(BlockStatement node)
   at NUglify.JavaScript.Visitors.OutputVisitor.OutputFunctionArgsAndBody(FunctionObject node)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(FunctionObject node)
   at NUglify.JavaScript.Syntax.FunctionObject.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.AcceptNodeWithParens(AstNode node, Boolean needsParens)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(BinaryExpression node)
   at NUglify.JavaScript.Syntax.BinaryExpression.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(AstNodeList node)
   at NUglify.JavaScript.Syntax.AstNodeList.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(BinaryExpression node)
   at NUglify.JavaScript.Syntax.BinaryExpression.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.Visit(BlockStatement node)
   at NUglify.JavaScript.Syntax.BlockStatement.Accept(IVisitor visitor)
   at NUglify.JavaScript.Visitors.OutputVisitor.Apply(TextWriter writer, AstNode node, CodeSettings settings)
   at NUglify.Uglify.Js(String source, String fileName, CodeSettings codeSettings)
   at WebOptimizer.JavaScriptMinifier.ExecuteAsync(IAssetContext config)
   at WebOptimizer.Asset.ExecuteAsync(HttpContext context, IWebOptimizerOptions options)
   at WebOptimizer.AssetBuilder.BuildAsync(IAsset asset, HttpContext context, IWebOptimizerOptions options)
   at WebOptimizer.AssetMiddleware.HandleAssetAsync(HttpContext context, IAsset asset, WebOptimizerOptions options)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)

**Comment by trullock** [comment]
Can you isolate and provide the offending input javascript which is causing this?

**Comment by afranz33** [comment]
Yes, the signalr.js file I downloaded from the cdn. Found here: https://cdn.jsdelivr.net/npm/@microsoft/signalr@6.0.2/dist/browser/signalr.js

**Comment by trullock** [comment]
I need a minimum repo to look at this please, not a whole file

**Comment by trullock** [comment]
I need the minimum amount of javascript that causes this.

Whilst keeping syntatic (but not functional) correctness, remove blocks of signalr.js until your exception goes away. That last bit you deleted contains the quirk. Put that block back and remove the rest. Repeat until you have the minimum amount of JS that causes this error and I can help you

**Comment by bstansbury-sds** [comment]
@trullock i have narrowed it down to the following code that is giving the error. mainly the `new.target`

```
class n extends Error {
  constructor() {
    const s = new.target.prototype;
  }
}
````

I hope this helps

**Comment by trullock** [comment]
TIL that is a syntax!

Yeah thats probably broken

Thanks for the repro!

**Comment by trullock** [comment]
fixed in 1.19.2
